lms/djangoapps/courseware/tests/test_i18n.py

```diff
@@ -47,6 +54,26 @@ def release_languages(self, languages):
             enabled=True
         ).save()
 
+    def create_user(self):
+        """
+        Creates the user log in
+        """
+        # Create one user and save it to the database
+
+        self.user = UserFactory.build(username='test', email=self.email, password=self.pwd)
```

Comment:
Nit: I don't think that email needs to be a class variable (I don't see that it is used anyplace else).